extra/reset-password.js

```diff
@@ -27,8 +27,17 @@ const main = async () => {
             console.log("Found user: " + user.username);
 
             while (true) {
-                let password = await question("New Password: ");
-                let confirmPassword = await question("Confirm New Password: ");
+                let password;
+                let confirmPassword;
+
+                # When called with "--new_password" argument for unattended modification (e.g. npm run reset-password -- --new_password=secret) 
+                if ('new_password' in args) {
+                    console.log("Note: Using password from argument");
```

Comment:
nit: other log statements don't use prefixes

```suggestion
                    console.log("Using password from argument");
```

Reply:
Prefix was removed from log statement.